src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java

```diff
@@ -0,0 +1,46 @@
+package ch.uzh.ifi.hase.soprafs24.controller;
+
+import ch.uzh.ifi.hase.soprafs24.entity.Session;
+import ch.uzh.ifi.hase.soprafs24.rest.dto.PlayerDTO;
```

Comment:
I think this line was forgotten to be taken out. or the dto wasn't commited. Is an invalid import that fails the build.

Reply:
+1